dis_001_significant_earthquakes/README.md

```diff
@@ -1,9 +1,7 @@
 ## Earthquakes Dataset Near Real-time Script
 This file describes the near real-time script that retrieves and processes the [Earthquake Catalog dataset](https://earthquake.usgs.gov/earthquakes/) for [display on Resource Watch](https://resourcewatch.org/data/explore/dis001nrt-Significant-Earthquakes-over-the-past-30-Days).
 
-{Describe how the original data came from the source.}
-
-{Describe the steps used to process the data, e.g., "convert variable X from the original netcdf file to a tif to upload to Google Earth Engine."}
+This dataset was provided by the source as a geojson file. Minimum significance of the earthquake was set to 0 to include all earthquake events. The data shown on Resource Watch can be found in the 'mag' variable of the 'properties' feature of the geojson. Datetime was retrieved from the 'time' variable in the 'properties' feature and was divided by 1000. The resulting table was uploaded to Carto.
```

Comment:
```suggestion
This dataset was provided by the source as a GeoJSON file. This GeoJSON was transformed into a table so that it could be uploaded to Carto. In order to transform the data from the GeoJSON to the data table used by Resource Watch, the following changes were made:
- The latitude and longitude in the GeoJSON's geometry were used to create the geometry shown on Resource Watch.
- The depth portion of the GeoJSON's geometry was separated into its own column in the Carto table.
- The data was provided by the source in the units of milliseconds since the Unix Epoch (the 'time' variable in the 'properties' feature). This was converted to a datetime object.
- A unique ID for each event was created based on the latitude, longitude, depth, and time. This was stored in a new column called 'uid'.

The data shown on Resource Watch can be found in the 'mag' variable of the 'properties' feature of the GeoJSON.
```